package/CHANGELOG

```diff
@@ -86,6 +86,7 @@ Fixes
   * Contact Analysis class respects PBC (Issue #2368)
 
 Enhancements
+  * Added computation of Mean Squared Displacements (#2438, PR #2619)
```

Comment:
Add your GitHub handle on the CHANGELOG for this version.

Reply:
I think its in there already :)